src/app/containers/ChartbeatAnalytics/amp/index.jsx

```diff
@@ -21,36 +16,14 @@ const JsonInlinedScript = data => (
   />
 );
 
-const AmpChartbeatBeacon = ({
-  domain,
-  type,
-  sections,
-  cookie,
-  chartbeatUID,
-}) => (
+const AmpChartbeatBeacon = ({ chartbeatConfig }) => (
   <amp-analytics type="chartbeat">
-    {JsonInlinedScript(
-      chartbeatAmpConfigOptions({
-        domain,
-        type,
-        sections,
-        cookie,
-        chartbeatUID,
-      }),
-    )}
+    {JsonInlinedScript(chartbeatAmpConfigOptions(chartbeatConfig))}
   </amp-analytics>
 );
 
 AmpChartbeatBeacon.propTypes = {
-  domain: string.isRequired,
-  type: string.isRequired,
-  sections: string.isRequired,
-  cookie: string,
-  chartbeatUID: number.isRequired,
-};
-
-AmpChartbeatBeacon.defaultProps = {
-  cookie: null,
+  chartbeatConfig: shape({}).isRequired,
```

Comment:
Can we be anymore specific on the expected object that `chartbeatConfig` can be?